cfg/pgrapher/experiment/dune-vd/wcls-sim-drift-simchannel-nf-sp.fcl

```diff
@@ -0,0 +1,98 @@
+#include "services_dune.fcl"
+
+process_name: wclssim
+physics :{
+   producers: {
+      plopper : {
+        module_type : BlipMaker
+      }
+      tpcrawdecoder : {
+         module_type : WireCellToolkit
+         wcls_main: {
+            tool_type: WCLS
+            apps: ["Pgrapher"]
+            # apps: ["TbbFlow"]
```

Comment:
Just beware of the pathology for Pgrapher when single depos are input to a large graph.  TbbFlow even when restricted to 1 thread may be substantially faster.

Reply:
Thanks @brettviren! I will push a `SimDepoSetSource` update in larwirecell and update the configurations.